src/runtime/parallel/worker.ts

```diff
@@ -18,7 +18,10 @@ import { doesHaveValue, valueOrDefault } from '../../value_checker'
 import { IRuntimeOptions } from '../index'
 import { PredictableTestRunStopwatch, RealTestRunStopwatch } from '../stopwatch'
 import { duration } from 'durations'
+import { pathToFileURL } from 'url'
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { importer } = require('../../importer')
```

Comment:
What makes this need to be a require instead of an import?

Reply:
If this is the only comment holding this back from being merged and released, can I help in any way? I could really use ESM support in cucumber in a new project. It would allow me to write the code in ESM instead of CommonJS modules. So if I can help... Do you need someone to try it out with this line changed to use `import`?

Reply:
Resolved by allowing JS in the TypeScript config, and adding JSDoc-flavoured type definitions to the file.